app/src/main/java/ch/sdp/vibester/activity/AuthenticationActivity.kt

```diff
@@ -36,16 +52,20 @@ class AuthenticationActivity : AppCompatActivity() {
         setContentView(R.layout.activity_authentication)
 
         val gso = GoogleSignInOptions.Builder(GoogleSignInOptions.DEFAULT_SIGN_IN)
+            .requestIdToken("7687769601-qiqrp6kt48v89ub76k9lkpefh9ls36ha.apps.googleusercontent.com")
```

Comment:
this string seems to be magic, consider putting it in a val and giving it a meaningful name.